### PR TITLE
fix(admin): fix wrong http code for patch method

### PR DESCRIPTION
**Pull request by An-DJ**
### Description

The default http code of patch method to admin.resources is `201`. It should be corrected to `200`.

### Checklist

- [X] I have explained the need for this PR and the problem it solves
- [X] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [X] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by soulbird — APPROVED** [review]

**Review by tokers — DISMISSED** [review]

File: apisix/core/etcd.lua [hunk 1]
```diff
@@ -507,7 +507,7 @@ function _M.atomic_set(key, value, ttl, mod_revision)
         key = key,
         value = value,
     }
-    res.status = 201
+    res.status = 200
```
Comment (spacewander) [inline apisix/core/etcd.lua:510]: Is the default status already 200?

Comment (An-DJ) [reply apisix/core/etcd.lua:510]: IMHO yes. Only the PATCH method uses this function now.

Comment (spacewander) [reply apisix/core/etcd.lua:510]: So we don't need to reassign it at all

Comment (An-DJ) [reply apisix/core/etcd.lua:510]: The res.status is assigned here and is only called by `PATCH`. That seems reasonable, doesn't it?

**Review by spacewander — APPROVED** [review]

**Review by soulbird — APPROVED** [review]

**Pull request merged**